src/control/server/server.go

```diff
@@ -160,6 +160,8 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 	membership := system.NewMembership(log)
 	scmProvider := scm.DefaultProvider(log)
 	harness := NewIOServerHarness(log)
+	netDevClass := cfg.Servers[0].Fabric.NetDevClass
```

Comment:
Do we know for sure that len(cfg.Servers) > 0 at this point?

Reply:
Good point -- with the recent landing of discovery mode, I don't think this is a safe assumption.

Reply:
You are right.  I've changed it to address that.